### PR TITLE
cancan configuration.

### PR DESCRIPTION
**Pull request by CUBOFIG**
# Description
Cancan gem configuration and ability model.

This gem allows us to authorize users for Ruby on Rails, this configuration will allow us to restrict what resources a specific user can access.

 # Testing
It's correct installation can be seen in the Gemfile.

### CODE REVIEW

**Review by YilKanda — DISMISSED** [review]
LGTM!

**Review by jestradadeveloper — APPROVED** [review]
LGTM

**Review by JuanequeX — APPROVED** [review]
LGTM

File: Gemfile [hunk 1]
```diff
@@ -24,6 +24,7 @@ gem 'react-rails', '~> 2.6', '>= 2.6.1'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 gem 'inline_svg'
+gem 'cancan'
```
Comment (EduardoGHdez) [inline Gemfile:27]:
```suggestion
gem 'cancancan' 
```

CanCan is no longer under mantainance, most of its community has migrated to use CanCanCan

From CanCan Documentation: 
> This project is abandoned, see its successor: CanCanCan


https://github.com/CanCanCommunity/cancancan

Comment (EduardoGHdez) [inline Gemfile:27]:
```suggestion
gem 'cancancan' 
```

CanCan is no longer under mantainance, most of its community has migrated to use CanCanCan

From CanCan Documentation: 
> This project is abandoned, see its successor: CanCanCan


https://github.com/CanCanCommunity/cancancan

**Review by EduardoGHdez — CHANGES_REQUESTED** [review]

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by YilKanda — APPROVED** [review]
LGTM!

**Comment by EduardoGHdez** [comment]
You're welcome 👋🏽

**Pull request merged**